src/api_server/src/lib.rs

```diff
@@ -200,6 +200,11 @@ impl ApiServer {
             _ => None,
         };
 
+        let vmm_new_state = match *vmm_action {
+            VmmAction::Pause => "Paused".to_string(),
+            VmmAction::Resume => "Running".to_string(),
```

Comment:
Let's keep the mechanism unified and just handle every state change here:
```suggestion
            VmmAction::StartMicroVm => "Running".to_string(),
            VmmAction::Pause => "Paused".to_string(),
            VmmAction::Resume => "Running".to_string(),
```